monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/ResultsFolderController.java

```diff
@@ -49,9 +51,16 @@
     @RolesAllowed(SecurityRoles.VIEWER)
     @RequestMapping(method = RequestMethod.GET, produces = "application/json")
     @ResponseBody
-    public List<Map<String, String>> resultsFolderJson(@PathVariable("tenant") final String tenant) {
-        final TenantContext context = _tenantContextFactory.getContext(tenant);
+    public List<Map<String, String>> resultsFolderJson(@PathVariable("tenant") final String tenant,
+            @RequestParam(value = "not_before", required = false) final String timestamp) {
 
+        final TenantContext context = _tenantContextFactory.getContext(tenant);
+        final Date searchedTimestamp;
+        if (timestamp == null || Long.valueOf(timestamp) < 0) {
+            searchedTimestamp = null;
+        } else {
+            searchedTimestamp = new Date(Long.valueOf(timestamp));
+        }
```

Comment:
that timestamp is null if the user wants all the results. Therefore I do not want to go the eligible method if I do not care about the timestamp